### DISCUSSION TITLE
Disable Markdown and newlines parsing/replacement: it breaks formatting

### DISCUSSION THREAD

**Discussion by MattPD** [ankitpokhrel/jira-cli#463] [Q&A]
Currently, when creating an issue using `--template`, triple backticks \`\`\` (in the template file used as input) surrounded by newlines are replaced with `{code]` without newlines.

For example, the original:
a
\`\`\`
b
c
d
\`\`\`

is replaced with

`a{code}bcd`

The problem is that the newline originally present after `a` is missing. The replacement of \`\`\` with `{code}` is also undesirable.
The reason for these two issues being a problem for the particular Jira instance is that it already has a Markdown plugin, so this breaks the formatting for created Jiras (and the missing newline makes code portions unreadable).

Is there a way to pass the raw input in template file as-is, without any changes or replacements?

Edit: FWIW, I've also tried using `[code}` directly in the input template file, but that turned out to be even worse: The rewrites now introduce `{code}` after several blank lines in the source code seemingly at random (?)--I wasn't able to determine the pattern thus far. For now I'm using triple backticks \`\`\` and manually re-inserting the newlines in the web editor, hopefully there's a better way...

**Comment by ankitpokhrel** [comment]
Hi @MattPD,

This is how the input you mentioned is converted in my installation (v8.22.1). (which looks correct)

![Screen Shot 2022-09-11 at 9 05 21 AM](https://user-images.githubusercontent.com/2364546/189516295-bb78e0a8-e868-4569-97a5-ea703f3c60d0.png)
The problem here seem to be because of the conflict with Markdown plugin you are using. I think it wouldn't hurt to have an option in jira-cli to skip processing the template file when creating/updating the issue. Will see what I can do there.

**Comment by MattPD** [comment]
Interesting, that does show `a` and `b` on the separate lines, indeed.
FWIW, the plugin is called "Markin" (I think it's <https://marketplace.atlassian.com/apps/1214558/markdown-for-jira-with-latex-and-uml-markin>).
Skipping the processing would help, thanks!

**Comment by jeffreysandoval** [comment]
I'm hitting this issue, too, using the same "Markin" plugin.  An option to skip the preprocessing would be great, I think.  I've been supplying the "description" field from stdin rather than a template file, if that makes a difference.
